### DISCUSSION TITLE
Mosquitto doesn't initialize properly

### DISCUSSION THREAD

**Discussion by stewe93** [simonsickle/unraid-templates#4] [Q&A / Support]
Hello There! 

First of all, thanks for maintaining this template! 
But I have a problem: the container doesn't start and produces the following error: 
```
text  error  warn  system  array  login  

1679006834: Error: Unable to open config file /mosquitto/config/mosquitto.conf.
1679077912: Error: Unable to open config file /mosquitto/config/mosquitto.conf.

** Press ANY KEY to close this window ** 
```

It doesn't start with the default config and doesn't copy the default to the mounted location.

Unraid server Basic, version 6.11.5
Docker version: 20.10.21

Can you help me with this?

**Comment by itslennie** [comment]
I resolved this by uploading a config.yaml file and changing the settings

config : /mnt/user/appdata/mosquitto/config

to

/mnt/user/appdata/mosquitto/config/config.yaml

**Comment by JDIacobbo** [comment]
I had this same issue an ended up creating a blank `mosquitto.conf` in `/mnt/user/appdata/mosquitto/config` and then restarted the container. That seems to have fixed the issue.

**Comment by Gizmor43** [comment]
just as a note

I also added this to the mosquitto.conf to allow teslamate connect to mosquitto otherwise mosquitto is running in local only mode

listener 1883
allow_anonymous true

This way my teslamate docker was connecting again to mosquitto.
